src/entities/Proposal/routes.ts

```diff
@@ -9,6 +9,7 @@ import { Request } from 'express'
 import isEthereumAddress from 'validator/lib/isEthereumAddress'
 import isUUID from 'validator/lib/isUUID'
 
+import { validateAddress } from '../../back/routes/validations'
```

Comment:
Maybe move this to another folder? `back/helpers/validations`, `back/utils/validations`

Reply:
I'm not sure about this one. 

My logic was that if we are going to have all routes in `back/routes`, and this validations are only going to be used in there (because they validate route params and and throw RequestErrors), then it's ok to have them in there.
I'm open to having them somewhere else, as long as we don't use them to validate outside routes.

WDYT?